mysql-native/src/main/java/org/ballerinalang/mysql/Utils.java

```diff
@@ -59,34 +59,38 @@ public static long getTimeout(Object secondsDecimal) {
         return -1;
     }
 
-    public static void addSSLOptions(BMap sslConfig, BMap<BString, Object> options) {
-        if (sslConfig == null) {
+    public static void addSSLOptions(BMap secureSocket, BMap<BString, Object> options) {
+        if (secureSocket == null) {
             options.put(Constants.DatabaseProps.SSL_MODE, Constants.DatabaseProps.SSL_MODE_DISABLED);
         } else {
-            BString mode = sslConfig.getStringValue(Constants.SSLConfig.MODE);
-            if (mode.getValue().equalsIgnoreCase(Constants.SSLConfig.VERIFY_CERT_MODE)) {
+            BString mode = secureSocket.getStringValue(Constants.SecureSocket.MODE);
+            if (mode.getValue().equalsIgnoreCase(Constants.SecureSocket.VERIFY_CERT_MODE)) {
                 mode = Constants.DatabaseProps.SSL_MODE_VERIFY_CA;
```

Comment:
AFAIU, we simply set the passed SSL mode from Ballerina side into Java options. Therefore, can't we update the Ballerina constant `public const SSL_VERIFY_CERT = "VERIFY_CERT";` as `public const SSL_VERIFY_CA = "VERIFY_CA";` and remove this if condition?
@lahirusamith @daneshk

Reply:
@ldclakmal, Yes It seems we can. As per the [documentation ](https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-security.html). @daneshk WDYT?

Reply:
+1

Reply:
changed in https://github.com/ballerina-platform/module-ballerinax-mysql/pull/96/commits/45b458de1d5df35268be9b467362e48c0d0fcc17